scrape/manager.go

```diff
@@ -121,8 +121,9 @@ func (m *Manager) Run(tsets <-chan map[string][]*targetgroup.Group) error {
 }
 
 func (m *Manager) reloader() {
+	// TODO: OK?
```

Comment:
This may be useful to speed up tests, WDYT? (I don't see why we silently limit it before)

Reply:
I didn’t catch your comments earlier.
It seems that the default is already set to 5s https://github.com/prometheus/prometheus/blob/8eb9228af8c0ff291fdebe2fc195764b6b7b355c/cmd/prometheus/main.go#L447-L448.
I’m not sure if there’s a specific reason for capping the value in this manner. If it could potentially cause issues, it might be best to explicitly mention this.
Users are generally allowed to customize other parameters, so it would make sense to allow them to adjust this one as well, provided they understand the implications.

Reply:
Is this conversation resolved?  If so, please remove the `TODO`.